spec/models/course/requirement/achievement_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
 RSpec.describe Course::Condition::Achievement, type: :model do
-  it { is_expected.to act_as(:condition).class_name(Course::Condition.name) }
+  it { is_expected.to acts_as(:condition).class_name(Course::Condition.name) }
```

Comment:
Okay I think I'm getting confused.

The declaration is `acts_as` -- the class *acts_as* another.

The matcher `act_as` -- the object should *act_as* another.

So the matcher should be `act_as`, but the declaration should be `acts_as`... I think I'll revert this. Sorry @minhtule